KebPOS/DataAccess/KebabContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿using KebPOS.Models;
+using Microsoft.EntityFrameworkCore;
```

Comment:
Somehow the csproj wasn't changed in your PR, so the project doesn't build. Could you please fix that?

Reply:
Should be fixed now.